### ISSUE TITLE
Installation fails with pip dependency issue in django-ansible-base/django-split-settings

### ISSUE THREAD

**Issue by mensler** [ansible/awx#15484]
### Please confirm the following

- [X] I agree to follow this project's [code of conduct](https://docs.ansible.com/ansible/latest/community/code_of_conduct.html).
- [X] I have checked the [current issues](https://github.com/ansible/awx/issues) for duplicates.
- [X] I understand that AWX is open source software provided for free and that I might not receive a timely response.
- [X] I am **NOT** reporting a (potential) security vulnerability. (These should be emailed to `security@ansible.com` instead.)

### Bug Summary

Docker compose installation (`make docker-compose-build`) fails with the following error:
```
#21 141.9 ERROR: Cannot install django-ansible-base[jwt-consumer,rbac,resource-registry,rest-filters]==2024.8.30.0.dev7+g47daf79 and django-split-settings==1.0.0 because these package versions have conflicting dependencies.
#21 141.9 
#21 141.9 The conflict is caused by:
#21 141.9     The user requested django-split-settings==1.0.0
#21 141.9     django-ansible-base[jwt-consumer,rbac,resource-registry,rest-filters] 2024.8.30.0.dev7+g47daf79 depends on django-split-settings==1.2.0
```

### AWX version

24.6.1

### Select the relevant components

- [ ] UI
- [ ] UI (tech preview)
- [ ] API
- [ ] Docs
- [ ] Collection
- [ ] CLI
- [X] Other

### Installation method

docker development environment

### Modifications

no

### Operating system

Amazon Linux 2023

### Steps to reproduce

Start Docker compose setup with `make docker-compose-build`.

### Expected results

Build successful.

### Actual results

Build fails after `RUN cd /tmp && make requirements_awx`with the pip dependency error, full output attached below.

### Additional information

[awx-installation-output.txt](https://github.com/user-attachments/files/16838038/awx-installation-output.txt)

**Comment by fosterseth** [comment]
for docker-compose environment, you will want to be on devel, not a tagged version of awx

**Comment by kpi88** [comment]
Perhaps the [compose README](https://github.com/ansible/awx/tree/devel/tools/docker-compose#clone-the-repo) needs updating, then. I would note also that building on the 24.6.1 tag was working until this weekend.

**Comment by fosterseth** [comment]
yeah the issue is that some of our requirements are tracking a devel branch of django ansible base library. When 24.6.1 was first released, it probably worked fine. But then more commits were added to django ansible base that broke with awx 24.6.1

Yeah I agree the docs should be instructing folks to use devel, not the latest tagged release.

**Comment by MalfuncEddie** [comment]
It is a bit worry some that the last tagged release of AWX before the re-architecture is kinda broken since it uses django-ansible-base  devel branch that is getting updated with "breaking" fixes of the rearchtecture. So not only a freeze in release that I quite understand but also breaking the last tagged release. This feels a bit wrong


Also anyone looking for a solution

edit requirements_git.txt to not install from the develbranch but from eg: 2024.8.28 branch
